docs/spark/clusters.md

```diff
@@ -1,6 +1,6 @@
 # Running Spark on CERN Clusters
 
-SWAN also allows to submit computations to Spark Clusters located at CERN. If you are interested in accessing these clusters, please follow the procedure indicated in the [Hadoop User Guide](https://hadoop-user-guide.web.cern.ch/getstart/access.html), as you need to be added to a specific e-group.
+SWAN also allows to submit computations to Spark Clusters located at CERN. If you are interested in accessing these clusters, please follow the procedure indicated in the [Hadoop User Guide](https://hadoop-user-guide.web.cern.ch/hadoop/hadoop/), as you need to be added to a specific e-group.
```

Comment:
I believe the right link that replaces the one broken is: https://hadoop-user-guide.web.cern.ch/getstart/access/

Reply:
It has been changed!